modules/standard/GMP.chpl

```diff
@@ -17,25 +17,157 @@
  * limitations under the License.
  */
 
-/* GMP Module 
+/*
+
+This module supports integration with the GMP library (the GNU Multiple
+Precision Arithmetic Library. See the `GMP homepage <https://gmplib.org/>`_
+for more information on this library.
+
+This README describes a prototype implementation of a standard GMP (GNU
```

Comment:
Maybe switch "This README describes a prototype..." to "This [module] is a prototype..."?